src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -245,10 +244,10 @@ export class ViolationTicketService {
 
     // special handling
     if (isDateFound || isTimeFound) {
-      result.issued_date = this.datePipe.transform(result[this.ocrTicketDateKey] + " " + result[this.ocrTicketTimeKey], "YYYY-MM-ddTHH:mm:ss");
```

Comment:
"YYYY-MM-ddTHH:mm:ssZ"

Reply:
results in a date not accepted by C# rest api